src/components/InstructorsOnly/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import {instructorsOnlyDescriptionText} from 'utils/text'
```

Comment:
These dumb text modules will be replaced with a real localization lib soon 😆 Only thing in the app that feels gross right now IMO.